substrate/client/rpc-spec-v2/src/chain_head/event.rs

```diff
@@ -35,13 +37,108 @@ pub struct ErrorEvent {
 /// This event is generated for:
 ///   - the first announced block by the follow subscription
 ///   - blocks that suffered a change in runtime compared with their parents
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase")]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(
+	rename_all = "camelCase",
```

Comment:
nit: we could use the `#[serde(rename_all = "camelCase")]` to `RuntimeVersionEventWrapper` instead and remove it from here